content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+
+- Istio with sidecar
+- Ambient with L4 ztunnel only
+- Ambient with L4 ztunnel and L7 waypoint proxies (one per namespace, scaled to 3 replicas)
+
+Spoilers! There are significant savings across the board for all ambient scenarios when compared to sidecars – up to 99% savings in usage and 98% in allocation.
+
+{{< image width="100%"
+    link="savings.png"
+    caption="Total Memory and CPU: Consumption and Allocation"
+    >}}
+
+Looking at total CPU and memory consumption, we have to remember that in the [ambient architecture](/blog/2022/introducing-ambient-mesh/), there are not sidecar containers for every application pod in the mesh. The result is that memory usage of Istio’s dataplane in the ztunnel-only ambient scenario uses *less than 1%* of what is used in sidecar scenarios, and still only 12% when waypoints are added. Looking at CPU, ztunnel once again uses 18% of what the sidecar scenario requires, and 60% when waypoints are deployed.
+
+Moving on to allocation, every sidecar resource has a default request of 100 millicores vCPU and 128 MB memory.  Assuming ztunnels and waypoint proxies have similar requests and limits as sidecars, that’s a *98% reduction* in allocated resources between L4 ambient and sidecar and 90% with waypoints!
+
+## Where the Savings Are Coming From
+
+Ambient mesh was designed to minimize resource requirements for users in their Kubernetes clusters. To explain how ambient does this, we must first clarify allocation versus utilization. When deploying a Kubernetes cluster on a hosted environment, nodes determine the overall capacity of the cluster and customer deployments, and pods are an allocation of that capacity. Utilization is a measure of how well this is done. As an architecture, sidecars interfere with effective utilization as they:
+
+- Define a high minimum for allocation at any scale
+- Can strand capacity by reserving more than is needed
+- Shift operational burden to adjust allocation if more than the default is needed
+
+The sidecar architecture forces this mode of allocation on users. Ambient solves these issues by leveraging a new architecture that separates the responsibilities of zero-trust networking and L7 policy handling. This is done with two new components to Istio: ztunnels and waypoint proxies.
+
+- [Ztunnels](/blog/2023/rust-based-ztunnel/) are a brand new Istio component written in Rust that are designed to be fast, secure, and lightweight. Ztunnels are deployed per node on a cluster and enable the most basic service mesh configurations for L4 features such as mTLS, telemetry, authentication, and L4 authorizations.
```

Comment:
key point to emphasize from resource saving perspective is ztunnel serves all pods co-located on the node